frontend/src/mocks/msw/handlers/user.ts

```diff
@@ -34,18 +32,6 @@ export const getUser = ({
   return rest.get<never, never, UserDto | ErrorDto>(
     '/api/v3/user',
     (_req, res, ctx) => {
-      // Check if the user is authenticated in this session
-      const isAuthenticated = localStorage.getItem(LOGGED_IN_KEY)
-      if (!isAuthenticated) {
-        // If not authenticated, respond with a 403 error
-        return res(
-          ctx.delay(),
-          ctx.status(403),
-          ctx.json({
-            message: 'Not authorized',
-          }),
-        )
-      }
       // If authenticated, return a mocked user details
       return res(ctx.delay(delay), ctx.status(200), ctx.json(mockUser))
     },
```

Comment:
If the mock getUser function no longer handles the case where the user is not authenticated, won't there be no necessity for the return type to be `UserDto | ErrorDto` because it will always be `UserDto`?